src/xdocs/report_issue.xml

```diff
@@ -99,16 +100,49 @@ private static final int SOMETHING = 1;
 Starting audit...
 Audit done.
 
+Expected: violation on first line.
+/var/tmp/Test.java:1:28: error: '{' is not preceded with whitespace.
         ]]></source>
       </p>
 
       <p>
-        Always specify what is expected.<br/>
+        For Windows users, please use "type" command instead of "cat".
+        Example of report that we expect:
 
-        <code>
-          Expected: violation on first line.
-          /var/tmp/Test.java:1:28: error: '{' is not preceded with whitespace.
-        </code>
+        <source><![CDATA[
+Check documentation: https://checkstyle.org/config_misc.html#Indentation
+
+D:\>type config.xml
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+          "https://checkstyle.org/dtds/configuration_1_3.dtd">
+<module name="Checker">
+    <module name="TreeWalker">
+        <module name="Indentation"/>
+    </module>
+</module>
+
+D:\>type Test.java
+class Test {
+    void method() {
+        try { return; // false positive
+        }
+        catch (Exception e) { return; // false positive
+        }
+    }
+}
+
+D:\>java -jar -Duser.language=en -Duser.country=US checkstyle-8.XX-all.jar-c config.xml Test.java
```

Comment:
same here

Reply:
I'm a bit more hesitant to change this one because the original linux report example also spells out the command in full without using `$RUN_LOCALE`. It should be consistent across both examples.

![image](https://user-images.githubusercontent.com/53135010/79068373-0ae98b00-7cf9-11ea-88f1-94b01ff1c2b3.png)

https://checkstyle.org/report_issue.html#How_to_report_a_bug.3F

Reply:
https://github.com/checkstyle/checkstyle/blob/master/.github/ISSUE_TEMPLATE.md

is our primary and most tunes format. We should align all to use it.
`RUN_LOCALE` was introduced for a reason to make command line short in width.
wihtout it, all valuable content appear in hidden are (you need to do horizontal scroll to get it) - it is pain and problem.
All should be visible wihtout any scrolling to see all details in the same time.

Please update linux example to follow https://github.com/checkstyle/checkstyle/blob/master/.github/ISSUE_TEMPLATE.md

Reply:
> Please update linux example to follow https://github.com/checkstyle/checkstyle/blob/master/.github/ISSUE_TEMPLATE.md

Fixed